composer.json

```diff
@@ -37,7 +37,9 @@
         "wilr/silverstripe-googlesitemaps": "~2.1.5",
         "silverstripe/theme-colorpicker": "^1",
         "silverstripe/theme-fontpicker": "^1",
-        "tractorcow/silverstripe-fluent": "^4.2"
+        "tractorcow/silverstripe-fluent": "^4.2",
+        "symfony/process": "^3.2",
```

Comment:
What are you using `symfony/process` for?

Reply:
* fulltextsearch needs symfony/process ^3
* framework needs symfony/process ^whatever
* ^whatever installs 4
* fulltextsearch cannot be installed
* explicitly use previous version
* no one complains